### PR TITLE
improve script listing to handle paths that end with / and those whic…

### PR DESCRIPTION
**Pull request by atlas0fd00m**
…h don't

### CODE REVIEW

File: envi/cli.py [hunk 1]
```diff
@@ -84,13 +84,16 @@ def getRelScriptsFromPath(scriptpaths):
...
-        baselen = len(basedir) + 1
+        baselen = len(basedir)
 
         for dirname,subdirs,subfiles in os.walk(basedir):
             for subfile in subfiles:
                 subpath = os.path.join(dirname,subfile)
                 if isValidScript(subpath):
-                    scripts.append(subpath[baselen:])
+                    script = subpath[baselen:]
+                    if script.startswith('/'):
```
Comment (invisig0th) [inline envi/cli.py]: windows?

Comment (atlas0fd00m) [inline envi/cli.py]: oh my.  derp!

Comment (atlas0fd00m) [inline envi/cli.py]: ummm... i repent in dust and ashes.  
PR fixed.  please re-review.

**Comment by atlas0fd00m** [comment]
please review again. <sheepish grin>

**Pull request merged**